graphql/totalReviewsByProductId1.graphql

```diff
@@ -0,0 +1,3 @@
+query TotalReviewsByProductId1($productId: String!) {
```

Comment:
No need to create new queries, just add a new field `ratingHistogram` in ReviewsResult.